src/utils/routes.ts

```diff
@@ -0,0 +1,9 @@
+export default {
+  ENUMS: {
+    CUISINE_TYPES: '/enums/CuisineType',
+    DAYS_OF_WEEK: '/enums/Day',
+    DISCOUNT_TYPES: '/enums/DiscountType',
+    PROMOTION_TYPES: 'enums/PromotionType',
+  },
+  PROMOTIONS: '/promotions',
+};
```

Comment:
Awesome I like this a lot, will make it much easier to know where all the routes are at. 

What do you think about adding in the rest of the routes, for e.g.
```
USERS: '/users',
DISCOUNTS: '/discounts', <-- don't think we'll be using this much
SAVED_PROMOTIONS: 'saved_promotions`,
SCHEDULES: 'schedules` <-- this one too
```

Reply:
Yeah good idea, I can add these!